azure-monitor-opentelemetry/setup.py

```diff
@@ -84,6 +84,7 @@
     },
     python_requires=">=3.7",
     install_requires=[
+        "azure-core-tracing-opentelemetry~=1.0.0b9",
```

Comment:
Should we pin this for now while that package is still in beta in case of breaking changes?

Reply:
Can do.